hi-ml-cpath/src/health_cpath/utils/output_utils.py

```diff
@@ -348,7 +348,7 @@ def save_validation_outputs(self, epoch_results: EpochResultsType, metrics_dict:
 
             # Writing completed successfully; delete temporary back-up
             if self.previous_validation_outputs_dir.exists():
-                shutil.rmtree(self.previous_validation_outputs_dir)
+                shutil.rmtree(self.previous_validation_outputs_dir, ignore_errors=True)
```

Comment:
why the need to ignore errors?

Reply:
Because of this hidden file ".fuse_hidden000" that causes rmtree to fail [AML ](https://ml.azure.com/experiments/id/a48fb202-54da-43d7-aabd-53728da8cd07/runs/TilesPandaImageNetMIL_1659515935_a1627f11?wsid=/subscriptions/a85ceddd-892e-4637-ae4b-67d15ddf5f2b/resourceGroups/health-ml/providers/Microsoft.MachineLearningServices/workspaces/hi-ml&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#outputsAndLogs)